pkg/localcontroller/storage/storage.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2021 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"path"
+	"path/filepath"
+
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/sedna/pkg/localcontroller/util"
+)
+
+const (
+	// S3Prefix defines that prefix of url is s3
+	S3Prefix = "s3"
+	// LocalPrefix defines that prefix of url is local host
+	LocalPrefix = ""
+	// S3EndPoint is s3 endpoint of the storage service
+	S3Endpoint = "s3-endpoint"
```

Comment:
THESE constant values should be shared with GM also, so need to find other common place

Reply:
S3Endpoint 
AccessKeyID 
SecretAccessKey